DuckDuckGo/SettingsRootView.swift

```diff
@@ -129,19 +125,11 @@ struct SettingsRootView: View {
                                                            onDisappear: {})
         case .duckPlayer:
             SettingsDuckPlayerView().environmentObject(viewModel)
-        default:
-            EmptyView()
-        }
```

Comment:
I've removed the possibility for the `netP` case to be provided to this function and the app then navigating to an empty view.